package.json

```diff
@@ -92,7 +96,7 @@
         "lodash": "^4.17.20",
         "maplibre-gl": "^2.0.0",
         "matrix-encrypt-attachment": "^1.0.3",
-        "matrix-events-sdk": "2.0.0",
+        "matrix-events-sdk": "0.0.1",
```

Comment:
That updates looks a bit odd to me, is that expected we downgrade to that version of the `events-sdk` ?

Reply:
Yes. That is causing the build to break. Matrix-events-SDK updates to ajv v8 which breaks ajv-keywords v3